tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -463,9 +463,9 @@ generic_config_updater/test_ecn_config_update.py::test_ecn_config_updates:
 
 generic_config_updater/test_eth_interface.py::test_replace_fec:
   skip:
-    reason: 'replace_fec ethernet test is not supported on virtual switch'
+    reason: 'replace_fec ethernet test depends on StateDB values. GCU needs to be updated to consider StateDB'
```

Comment:
Previously I opened PR #8601 to skip this test on `M0` and `Mx` topo and it's merged yesterday. That PR will cause a merge conflict here. As current code says, replace fec is not supported on virtual switch, should we keep it skipped for this scenario?

Reply:
I changed this to skip `test_replace_fec` on all platforms for now because GCU should block the change on platforms where `fec` is not supported. So the new code skips this test on all platforms until GCU is updated to block the change on platforms where `fec` is not supported. Then, the plan is to remove this skip and run `test_replace_fec` on all platforms. On platforms where `fec` is not supported, it should expect failure from GCU